src/app/header/header.component.html

```diff
@@ -0,0 +1,16 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a routerLink="/" class="navbar-brand">Toggl</a>
+    </div>
+    <div class="collapse navbar-collapse">
+      <ul class="nav navbar-nav navbar-right">
```

Comment:
Links should be generated dynamically (use *ngFor)

Reply:
done